mongodbatlas/table_mongodbatlas_container.go

```diff
@@ -0,0 +1,182 @@
+package mongodbatlas
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+func tableMongoDBAtlasContainer(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "table_mongodbatlas_container",
+		Description: "Containers in a `mongodbatlas` project allows for cloud provider backed virtual private networking - dubbed as `container network peering` in MongoDB Atlas",
```

Comment:
```suggestion
		Description: "Containers in a project allows for cloud provider backed virtual private networking - dubbed as container network peering in MongoDB Atlas",
```

These descriptions aren't shown on the Hub site (instead the Markdown docs are), so any Markdown formatting, like using backticks, should not be included in table descriptions.

Can you please remove other instances of Markdown formatting in other table `Description` fields as well?